cmd/cstor-pool-mgmt/pool/v1alpha2/status.go

```diff
@@ -0,0 +1,105 @@
+package v1alpha2
+
+import (
+	"strings"
+
+	api "github.com/openebs/maya/pkg/apis/openebs.io/v1alpha2"
+	zfs "github.com/openebs/maya/pkg/zfs/cmd/v1alpha1"
+)
+
+const (
+	// StatusNoPoolsAvailable .. not pools available
+	StatusNoPoolsAvailable = "no pools available"
+	// PoolStatusDegraded .. pool is degraded
+	PoolStatusDegraded = "DEGRADED"
+	// PoolStatusFaulted .. pool is in faulty state
+	PoolStatusFaulted = "FAULTED"
+	// PoolStatusOffline .. pool is in offline state
+	PoolStatusOffline = "OFFLINE"
+	// PoolStatusOnline .. pool is online
+	PoolStatusOnline = "ONLINE"
+	// PoolStatusRemoved .. pool is in removed state
+	PoolStatusRemoved = "REMOVED"
+	// PoolStatusUnavail .. pool is unavailable
+	PoolStatusUnavail = "UNAVAIL"
+)
+
+// GetStatus return status of the pool
+func GetStatus(csp *api.CStorNPool) (api.CStorPoolStatus, error) {
```

Comment:
return supposed to be CStorNPoolStatus?

Reply:
I kept internal struct as CStorPool, since we are going to replace `CStorNpool` as `CStorPool`.